dashboard/app/models/pd/enrollment_notification.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: pd_enrollment_notifications
+#
+#  id               :integer          not null, primary key
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  pd_enrollment_id :integer          not null
+#  name             :string(255)
+#
+# Indexes
+#
+#  index_pd_enrollment_notifications_on_pd_enrollment_id  (pd_enrollment_id)
+#
+
+class Pd::EnrollmentNotification < ActiveRecord::Base
+  belongs_to :enrollment, class_name: 'Pd::Enrollment', foreign_key: :pd_enrollment_id
+end
```

Comment:
I'm fine with calling it `enrollment` or `pd_enrollment`, whatever you prefer to have it called in the rest of your code. if you're having trouble deciding, then `enrollment` sounds fine since the context should make it obvious that its a pd enrollment.